src/Common/src/System/Net/Security/CertificateHelper.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
>it should probably be moved to a \Http folder to avoid a confusion

If it is used by ClientWebSocket, then I don't see how it makes sense to be in \Http folder.